### DISCUSSION TITLE
How to configure dependencies resolving

### DISCUSSION THREAD

**Discussion by jiramares** [VirtusLab/scala-cli#2762] [General]
Hi,
we are using local artifactory to resolve artifacts, is it possible to configure scala-cli to use our local artifactory? Or there is only possibility to pass `--repository` argument or put into every script `//> using repository`?
Thanks a lot

**Comment by adpi2** [comment]
@jiramares Did you try using the `COURSIER_REPOSITORIES` environment variable? It should work with the Coursier CLI and probably also with Scala CLI.

See documentation [here](https://get-coursier.io/docs/other-repositories).

**Comment by jiramares** [comment]
Thanks a lot that helps. Jirka
